tensorflow_addons/image/dense_image_warp_test.py

```diff
@@ -88,29 +87,23 @@ def test_unknown_shape():
         assert res.shape == (shape[0], 4, shape[3])
 
 
-@test_utils.run_all_in_graph_and_eager_modes
-class DenseImageWarpTest(tf.test.TestCase):
-    def _check_zero_flow_correctness(self, shape, image_type, flow_type):
-        """Assert using zero flows doesn't change the input image."""
-        rand_image, rand_flows = _get_random_image_and_flows(
-            shape, image_type, flow_type
-        )
-        rand_flows *= 0
+def _check_zero_flow_correctness(shape, image_type, flow_type):
+    """Assert using zero flows doesn't change the input image."""
+    rand_image, rand_flows = _get_random_image_and_flows(shape, image_type, flow_type)
+    rand_flows *= 0
 
-        interp = dense_image_warp(
-            image=tf.convert_to_tensor(rand_image),
-            flow=tf.convert_to_tensor(rand_flows),
-        )
+    interp = dense_image_warp(
+        image=tf.convert_to_tensor(rand_image), flow=tf.convert_to_tensor(rand_flows),
+    )
+
+    np.testing.assert_allclose(rand_image, interp, rtol=1e-3)
```

Comment:
```suggestion
    np.testing.assert_allclose(rand_image, interp, rtol=1e-6, atol=1e-6)
```

For info, the signature of assertAllClose is `def assertAllClose(self, a, b, rtol=1e-6, atol=1e-6, msg=None):`